src/cdomain/value/cdomains/int/intDomTuple.ml

```diff
@@ -82,7 +86,7 @@ module IntDomTupleImpl = struct
     let intv =  f p2 @@ r.fi2_ovc (module I2) in
     let intv_set = f p5 @@ r.fi2_ovc (module I5) in
     let bf = f p6 @@ r.fi2_ovc (module I6) in
-    ignore (check_ov ~suppress_ovwarn ~cast:false ik intv intv_set bf);
+    ignore (check_ov ~suppress_ovwarn ~no_safe:true ~cast:false ik intv intv_set bf);
```

Comment:
Was this only to suppress safe checks on integer literals? if so, then this might be redundant after #1794.

Reply:
But that's on this branch with the `~no_safe:true` in place? Because it would mean that the `~no_safe` isn't even silencing what it is trying to silence.